### PR TITLE
Redirect traffic to HTTPS from HTTP if not done automatically

### PR DESCRIPTION
**Pull request by kaustubhmallik**
PR for the issue "Automatically redirect http traffic to https" (#7 in open issues).  Created a new javascript file in src directory and added it in Gruntfile. Please have a look and suggest if anything is missing.

### CODE REVIEW

File: src/redirect_traffic.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// redirect traffic to HTTPS from HTTP
+if (location.hostname != 'localhost' && location.hostname != 'localhost' && location.protocol != 'https:') {
```
Comment (joshghent) [inline src/redirect_traffic.js]: You seem to have repeated the `location.hostname != 'localhost'` code

**Comment by joshghent** [comment]
Added a quick change that needs making above but other than that, looks good to go.

Thanks for your contribution! 🎉

File: src/redirect_traffic.js (hunk 1 above)
Comment (kaustubhmallik) [inline src/redirect_traffic.js]: oops sorry, actually I was thinking of checking it for localhost as well as for 127.0.0.1 Should I edit it, and send a new PR?

**Comment by kaustubhmallik** [comment]
Hey josh, I have corrected and committed it. As, its my first step in open-source, I am a bit nervous too. Please let me know if its good enough to be merged or should I add something more. Thanks.

**Comment by joshghent** [comment]
@kaustubhmallik Great work, thanks. Approved.

Look forward to your next PR. Congrats on your first 🎉

**Comment by kaustubhmallik** [comment]
Thanks a lot josh. I am working on two another PR. Will complete and send them asap. Thanks again 👍

**Comment by joshghent** [comment]
@kaustubhmallik https://github.com/esfiddle/esfiddle/blob/develop/CONTRIBUTORS.md 👍

**Pull request merged**